### DISCUSSION TITLE
Use a single field for startDate and startTime

### DISCUSSION THREAD

**Discussion by AtlcatlQ** [davish/obsidian-full-calendar#359] [Feature Requests]
Due to YAML supports ISO date and time format, I think could be useful to have the start date and time in the same variable like: `startDate: YYYY-MM-DDTHH:mm` And could be used also for endDate and endTime. I don't know if it's feasible or if take a lot of work.

If it's possible, it's also make work multiple day events.

I apreciate your work

**Comment by hittjwX** [comment]
Using a `startDate: ` with an ISO `YYYY-MM-DD HH:mm` would allow an `endDate: ` with an event spanning multiple days. A use case would be flagging a tentative five-day window for Vacation with a single entry. An ISO date would simplify other YAML front matter.

**Comment by davish** [comment]
@jwhco, you can create multi-day events by clicking and dragging on the calendar.
